packages/Spellcheck-Core.package/SpellcheckOrganizeUI.class/class/registerInWorldMenu.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+registerInWorldMenu
+	TheWorldMenu registerOpenCommand: {'SpellcheckOrganizeUI'. {self. #open}}
```

Comment:
You need to call this line/method from your workspace, because initialize (of the class) is only called when you install it (at least thats how I think it works).

Reply:
Yeah I noticed that too. For dev purposes I put this line in the initialize method of the instance but I'm fine with calling it once in the workspace